### PR TITLE
[IMP] practical/db_management: hosting changes

### PR DESCRIPTION
**Pull request by rim-odoo**
task 2185120

### CODE REVIEW

File: db_management/hosting_changes.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+:banner: banners/hosting_changes.png
```
Comment (AntoineVDV) [inline db_management/hosting_changes.rst:1]: This file doesn't seem to exist :p

Comment (jcs-odoo) [reply db_management/hosting_changes.rst:1]: @AntoineVDV  it still doesn't. Should I add one for my PR to backport this doc down to 12.0?
@rim-odoo did you have a picture for this + is it okay for you to backport to 12.0?

Cheers :)

Comment (AntoineVDV) [reply db_management/hosting_changes.rst:1]: yup

Comment (rim-odoo) [reply db_management/hosting_changes.rst:1]: I forgot to create/add a picture, and if you want to back port it to 12.0 you can (this part of the doc is not versioned)

**Pull request merged**